add-domain

```diff
@@ -491,3 +491,46 @@ cliphot-portugal-vip.xyz
 pdfkinghub.xyz
 stopcorona.xyz
 coronavirus.zone
+web-lgsupportforlive.com
+violatesolutioncenter.com
+kadinasidettehyr.ml
+objection-services.com
+ig.copyright.officiall.ml
+copyrighthelpforcontact.com
+sakirsevgili.tk
+sporcu.cf
+telifdogrulama-ig.com
+helponactives.ml
+web-lgsupportforlive.com
+int-user-contradiction.ml
+telifhakkifrom.ml
+verifydeneme23412.tk
+formcopyrightviolation.ga
+help-bluebadge.com
+telifhakkisorunumuz.ml
+siddetesessizkalma.ml
+bankaconfirmform.ml
+help-media.com
+caseaccount-media.tk
+feedbackformedia.ml
+feeds-supportivee.com
+ig.mictorys.ml
+client-helpersz.ml
+mybussineshelp.ml
+ig-chatservices.ml
```

Comment:
These two is to be removed

![image](https://user-images.githubusercontent.com/44526987/136653716-b912d32e-4451-4a01-907d-803d08275621.png)